docs/integrations/spark/configuration.md

```diff
@@ -0,0 +1,150 @@
+---
+sidebar_position: 2
+title: Configuration
+---
+
+Configuring the OpenLineage Spark integration is straightforward. It uses built-in Spark configuration mechanisms.
+
+Your options are:
+
+1. [Setting the properties directly in your application](#setting-the-properties-directly-in-your-application).
+2. [Using `--conf` options with the CLI](#using---conf-options-with-the-cli).
+3. [Adding properties to the `spark-defaults.conf` file in the `${SPARK_HOME}/conf` directory](#adding-properties-to-the-spark-defaultsconf-file-in-the-spark_homeconf-directory).
+
+#### Setting the properties directly in your application
+
+The below example demonstrates how to set the properties directly in your application when
+constructing
+a `SparkSession`.
+
+:::warning
+The setting `config("spark.extraListeners", "io.openlineage.spark.agent.OpenLineageSparkListener")` is
+**extremely important**. Without it, the OpenLineage Spark integration will not be invoked, rendering
+the integration ineffective.
+:::
+
+<Tabs groupId="spark-app-conf">
+<TabItem value="scala" label="Scala">
```

Comment:
Here i got the same, no tabs are rendered.

Reply:
missing import. Thanks for spotting this.